### ISSUE TITLE
Cannot read property 'price' of undefined

### ISSUE THREAD

**Issue by TriplEight** [w3f/staking-rewards-collector#47]
How do I debug this error?

```bash
$ yarn start
yarn run v1.22.18
$ node src/index.js
TypeError: Cannot read property 'price' of undefined
    at addPriceData (file:///home/tripleight/code/github.com/tripleight/staking-rewards-collector/src/api.js:20:76)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async gatherData (file:///home/tripleight/code/github.com/tripleight/staking-rewards-collector/src/gatherData.js:15:15)
    at async main (file:///home/tripleight/code/github.com/tripleight/staking-rewards-collector/src/index.js:48:11)
Done in 2.98s.
```

**Comment by jonasW3F** [comment]
That might happen if the coingecko API does not return a price but the script wants to access it. You want to narrow the issue down by checking with account causes this error (and which chain), then check the dates that you request prices for. If you want to share the userInput.json with me (sending it in private), I could take a look, but it would expose your account.

**Comment by jonasW3F** [comment]
@TriplEight Does this issue still persist for you? Otherwise, I'd close the issue.

**Comment by jonasW3F** [comment]
The issue should be fixed in https://github.com/w3f/staking-rewards-collector/pull/51 - Could you check again?

**Comment by jonasW3F** [comment]
Should be fixed with v1.7.0
